File: Strings/1168/1168.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int qtdTestes;
    int ledsNecessarios[10] = { 6, 2, 5, 5, 4, 5, 6, 3, 7, 6 };
    
    cin >> qtdTestes;
    while(qtdTestes--){
        char numeros [1000000];
        cin >> numeros;
        int total = 0;
        
        for(int i=0; numeros[i] != '\0'; i++) total += ledsNecessarios[numeros[i]-48];
        
        cout << total << " leds\n";
    }
    return 0;
}

File: Strings/1168/1168.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int qtdTestes = Integer.parseInt(scan.nextLine());
		byte[] ledsNecessarios = { 6, 2, 5, 5, 4, 5, 6, 3, 7, 6 };

		while (qtdTestes-- > 0) {
			char[] numeros = scan.next().toCharArray();
			int total = 0;
			
			for (int numero : numeros) total += ledsNecessarios[numero-48];
			System.out.println(total + " leds");
		}
	}
}